relay_handler.js

```diff
@@ -625,6 +644,14 @@ function _observeErrorFrame(errFrame, now) {
     var code = res.value;
     var codeName = v2.ErrorResponse.CodeNames[code] || 'unknown';
 
+    if (self.inreq && self.inreq.circuit) {
```

Comment:
self.inreq is not nullable, we do not have a preceding pattern of null checking it anywhere